src/main/kotlin/me/partlysanestudios/partlysaneskies/gui/hud/cooldown/Cooldown.kt

```diff
@@ -0,0 +1,65 @@
+//
+// Written by Su386.
+// See LICENSE for copyright and license notices.
+//
+
+
+package me.partlysanestudios.partlysaneskies.gui.hud.cooldown
+
+import me.partlysanestudios.partlysaneskies.PartlySaneSkies
+import me.partlysanestudios.partlysaneskies.utils.MathUtils
+import net.minecraft.item.ItemStack
+
+abstract class Cooldown {
```

Comment:
Does this need to be a class? Since your treecap thing is an object

Reply:
Yes

Reply:
You can't do an abstract object.

Reply:
I see